Tests/MarkdownKitTests/LocationTests.swift

```diff
@@ -10,4 +10,19 @@ final class LocationsTests: XCTestCase {
         let paragraph = document.children.first!
         XCTAssertEqual(markdown, paragraph.content)
     }
+
+    func testRangesWithConsecutiveHTMLComments() {
+        let markdown = """
+<!-- Here's an HTML comment -->
+<!-- and a second one with some more content -->
+"""
+        let document = Parser.parse(markdown)!
+        XCTAssertEqual(markdown, document.content)
+
+        let firstLine = document.children.first
+        let secondLine = document.children[document.children.startIndex.advanced(by: 1)]
+
+        XCTAssertNotNil(firstLine?.range)
+        XCTAssertNil(secondLine.range)
```

Comment:
Mind removing this or adding a comment? If we fix this cmark bug in the future, this test would fail.

Reply:
Oh ya, whoops. I started out with this test before writing the other one. I'll remove this whole test, tbh.